documentation/manual/helpers/OSHDBDriver.md

```diff
@@ -0,0 +1,79 @@
+# OSHDBDriver
+## Installation
+
+Replace your OSHDB dependency with the following:
+
+```xml
+<dependency>
+   <groupId>org.heigit.ohsome</groupId>
+   <artifactId>oshdb-database-driver</artifactId>
+   <version>0.7.2</version>
```

Comment:
Should we have in this docu a fix version? Maybe better a placeholder like:
```suggestion
   <version>LATEST_VERSION</version>
```

Reply:
That is a more general question than this PR. We copied the documentation from other places in the repository. Currently it is setup for all version references to be changed after a stable release according to our internal documentation.

Btw. I couldn't find any documentation on `LATEST_VERSION` in maven. Do you know on which specific release this will be pointing to?

Reply:
P.S.: I assume it would be pointing to either `1.0.0-SNAPSHOT ` or `1.0.0-beta-1` at the moment. Do we want people to use this version already?